src/traffic_server/traffic_server.cc

```diff
@@ -2094,7 +2116,11 @@ main(int /* argc ATS_UNUSED */, const char **argv)
       if (delay_p && ink_atomic_cas(&delay_listen_for_cache_p, 0, 1)) {
         Debug("http_listen", "Delaying listen, waiting for cache initialization");
       } else {
+        // Since the previous delay_listen_for_cache_p was not 0, then the
+        // cache must have been initialized already.
```

Comment:
"or ATS is not waiting for the cache to make the ports ready"

Reply:
I'm open to improving the comment. What are you fixing about it? Is it wrong to say that the cache must have been initialized by this point? As I read things, if we get to this branch then the value must have been -1 and thus the cache must be initialized and I'm trying to help the reader see that's the case.

Reply:
Alan pointed out offline that we should take delay_p, tied to the value of wait_for_cache, into consideration with this comment. I've updated the comment accordingly.

Thanks for the review Alan!